server/src/main/kotlin/com/xebia/functional/xef/server/http/routes/AIRoutes.kt

```diff
@@ -62,6 +62,7 @@ fun Routing.aiRoutes(
     }
 }
 
+@Deprecated("will be removed in future pr")
```

Comment:
We don't have any issue for that.  For me, this is not a valid description for a deprecated function